miden-lib/asm/miden/contracts/wallets/basic.masm

```diff
@@ -1,6 +1,10 @@
 use.miden::account
 use.miden::tx
 
+# CONSTANTS
+# =================================================================================================
+const.PUBLIC_NOTE=2
```

Comment:
AFAIK the constants exports is not yet implemented, so we have to copy the constants for the time being